$OpenBSD: patch-gst-libs_ext_libav_libavutil_Makefile,v 1.2 2014/10/07 16:06:09 ajacoutot Exp $

libavutil.a(log2_tab.o)(.rodata+0x0): multiple definition of `ff_log2_tab'
libavformat.a(log2_tab.o)(.rodata+0x0): first defined here

--- gst-libs/ext/libav/libavutil/Makefile.orig	Wed Sep 24 10:55:09 2014
+++ gst-libs/ext/libav/libavutil/Makefile	Tue Oct  7 17:20:02 2014
@@ -82,7 +82,6 @@ OBJS = adler32.o                                      
        lfg.o                                                            \
        lls.o                                                            \
        log.o                                                            \
-       log2_tab.o                                                       \
        mathematics.o                                                    \
        md5.o                                                            \
        mem.o                                                            \
